templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -32,6 +43,8 @@ contents:
     done
 
     if [ "$iface" = "br-ex" ]; then
+      intf=$(ovs-vsctl list-ports ${iface})
```

Comment:
is this good enough? how do we uniquely get the interface after br-ex is already configured? will there be other ports that br-ex might have?

Reply:
I think you want to do `ovs-vsctl list-ifaces br-ex` here. Then loop through the interfaces and call configure_driver_options on each one. The port could be a bond, with underlying interfaces attached to it. I'm not sure if that will ever happen in a vsphere deployment, but better to be safe than sorry.

Reply:
We currently aren't handling bond and vlans anyway during the initial bring-up right? If there were indeed sub-interfaces in the config that get moved to br-ex, yes, that makes sense.